packages/farcaster-js-hub-rest/src/openapi/post-process.sh

```diff
@@ -0,0 +1,3 @@
+sed 's/File/Buffer/g' src/openapi/generated/apis/submit-message-api.ts > src/openapi/generated/apis/submit-message-api-temp.ts
```

Comment:
The generated code for binary string is of type File, which is [not supported by version 3](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators/typescript-axios.md#data-type-feature). This script replaces the occurrences of `File` with `Buffer` to make this work.